app_api_template.coffee

```diff
@@ -10,6 +10,15 @@ startApp = ->
 	http = require 'http'
 	path = require 'path'
 
+	favicon = require('serve-favicon')
+	logger = require('morgan')
+	methodOverride = require('method-override')
+	session = require('express-session')
+	bodyParser = require('body-parser')
+	# multer = require('multer')
```

Comment:
It actually isn't being used as far as I can tell anywhere in ACAS.  Our forms typically upload a file first and then POST using `application/x-www-form-urlencoded;`.  Using multipart posts require you to configure an upload directory for the files and images uploaded in the multipart post.  While this is easy, we actually have never had this as a feature.  I can add it later if we want to.  The reason multer is even in this file is because I was following the migration procedure from express 3 -> 4 and they had it in there.  I realized we didn't need it so I commented it out.  I will remove the commented out line in a commit to resolve this.